regression/cuda/test_log_softmax.c

```diff
@@ -0,0 +1,175 @@
+#include "test_log_softmax.h"
```

Comment:
Could you copy the bsg_f1 license at the top of this file? You can find it at the top of all other files in bsg_f1.

Reply:
Done!